chart/kube-flannel/templates/config.yaml

```diff
@@ -2,7 +2,6 @@ kind: ConfigMap
 apiVersion: v1
 metadata:
   name: kube-flannel-cfg
-  namespace: kube-flannel
```

Comment:
I know that using `--namespace` will automatically add the resources on the configured namespace but I prefer to specify `namespace: {{ .Release.Namespace }}`

Reply:
Done